sky/data/mounting_utils.py

```diff
@@ -122,6 +123,11 @@ def get_az_mount_cmd(container_name: str,
     cache_path = _BLOBFUSE_CACHE_DIR.format(
         storage_account_name=storage_account_name,
         container_name=container_name)
+    # The line below ensures the cache directory is new before mounting to
+    # avoid "config error in file_cache [temp directory not empty]" error, which
+    # can occur after stopping and starting the same cluster on Azure.
+    # This helps ensure a clean state for blobfuse2 operations.
+    cache_path += f'_{int(time.time())}'
```

Comment:
Does this cause the same issue for `launch` on existing `UP` cluster where we create a new folder everytime?